src/lang/KclSingleton.ts

```diff
@@ -58,21 +59,29 @@ export class KclManager {
   private _execState: ExecState = emptyExecState()
   private _programMemory: ProgramMemory = ProgramMemory.empty()
   lastSuccessfulProgramMemory: ProgramMemory = ProgramMemory.empty()
+  lastSuccessfulOperations: Operation[] = []
   private _logs: string[] = []
+  private _errors: KCLError[] = []
   private _diagnostics: Diagnostic[] = []
   private _isExecuting = false
   private _executeIsStale: ExecuteArgs | null = null
   private _wasmInitFailed = true
   private _hasErrors = false
   private _switchedFiles = false
+  private _defaultPlanesVisibility = {
+    xy: true,
+    yz: true,
+    xz: true,
+  }
```

Comment:
@franknoirot just curious, where is this used?

Reply:
Maybe for future use?

Reply:
Nope from an abortive attempt to track the default planes visibility within this class. I'll remove it now, thanks for catching!

Reply:
Ofc! Ty